tasks/reduce-snapshot/reduce-snapshot.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: reduce-snapshot
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to reduce snapshot
+  params:
```

Comment:
In all our other tasks we have the descriptions listed for each param in the yaml

Reply:
added descriptions